### PR TITLE
Making SetModTime less scary by printing as Debugf.

### PR DESCRIPTION
**Pull request by TGion**
#### What is the purpose of this change?

Changed "SetModTime not supported" feature to print out the information from `Errorf` to `Debugf`.
Functionality was given before, so the message could be considered more of a warning.

It was / is observed with different ftp implementations - confirmed with the implementation of `FRITZ!Box 7530 - FRITZ!OS:7.51-104920 BETA`

#### Was the change discussed in an issue or in the forum before?

https://forum.rclone.org/t/ftp-fritz-box-setmodtime-is-not-supported/37781

#### Checklist

- [x] I have read the [contribution guidelines](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#submitting-a-new-feature-or-bug-fix).
- [x] I have added tests for all changes in this PR if appropriate.
- [x] I have added documentation for the changes if appropriate.
- [x] All commit messages are in [house style](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#commit-messages).
- [x] I'm done, this Pull Request is ready for review :-)

### CODE REVIEW

**Comment by olefrost** [comment]
I have approved the workflow.

@ncw can I just for fun try approving and merging this (if everything looks good to you)?

**Review by ncw — APPROVED** [review]
Looks great - thank you :-)

**Comment by ncw** [comment]
> I have approved the workflow.
> 
> @ncw can I just for fun try approving and merging this (if everything looks good to you)?

@olefrost yes that is a good idea.

Note that this commit needs its commit message editing, so do "Rebase and merge" and you'll get a chance to edit it. Don't leave the reference to the PR number - that isn't particularly helpful.

The first line of the commit message needs to be something appropriate for the changelog, and it should have a link to the forum post in the body.

Something like this - feel free to expand / rewrite

```
ftp: lower log message priority when SetModTime is not supported to debug

See: https://forum.rclone.org/t/ftp-fritz-box-setmodtime-is-not-supported/37781
```

If in doubt see [the docs on commit messages](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#commit-messages)

**Comment by ncw** [comment]
@TGion thank you for the pull request and I hope you enjoyed the process of getting there! I'll get @olefrost to touch up the commit message which is good practice for him :-)

**Comment by TGion** [comment]
> @TGion thank you for the pull request and I hope you enjoyed the process of getting there! I'll get @olefrost to touch up the commit message which is good practice for him :-)

Thanks Nick, i did!
And sorry for the commit message. Your changes make absolute sense and I will improve mine the next time.
There is really a lot more to learn here.
Thanks again for your patience. Take care!

@olefrost Thanks for your merge.

**Comment by ncw** [comment]
PS @olefrost updated above instructions to say "Squash and merge" which is what it should have said originally!

**Comment by olefrost** [comment]
@ncw Should there be a blank line in the top of the description, or can I place the "See:..." at the top?

**Comment by ncw** [comment]
@olefrost 
> Should there be a blank line in the top of the description, or can I place the "See:..." at the top?

There needs to be a blank line after the title line. I forget why though! So

1. ftp: ...
2.
3. See: ...

**Comment by olefrost** [comment]
@TGion 
Thanks for your contribution!
No need to excuse, you made a very fine contribution and I fully agree there is a lot to learn. 
[This](https://github.com/rclone/rclone/pull/5273) was my first contribution – and still learning as you see :)

@ncw
Thanks for letting me try, it was fun!
I think I will make a small toy project to get some practice before I dare merge something more complex into rclone :)

PS: Looks like the GitHub GUI automatically inserts the blank line in the commit message, so this commit got a little beginner signature from me.

**Comment by ncw** [comment]
No worries @olefrost about the extra blank line - it is very minor!

Do you think a bit in the https://github.com/rclone/rclone/blob/master/MAINTAINERS.md doc would be useful on best practices for reviewing and merging pull requests?

**Comment by olefrost** [comment]
Thanks @ncw

The guide is fine, no problem in rclone assuming some prior experience in managing pull requests from a smaller project.

I think it takes a certain amount of mistakes to get the necessary experience, and these mistakes are best made in a project where no/less users are directly affected. This was my first ever, I enjoyed the thrill :)

**Pull request merged**
